src/api/app/models/event/base.rb

```diff
@@ -1,3 +1,5 @@
+require 'rabbitmq_bus'
```

Comment:
Shouldn't this class be autoloaded by rails?

Reply:
That didn't seem to work. Let me retry

Reply:
No, it won't work directly... you need to add the path to the ```eager_load_paths``` like I did for the Backend one at https://github.com/openSUSE/open-build-service/blob/master/src/api/config/application.rb#L18

Reply:
Can we just eager load all of `/lib`? Or would this cause problems?

Reply:
No... I prefer to have this narrowed for now. Loading all the things there is like killing flies with a cannon 😄

Reply:
Well we load all the files in `/app` already.. but ok if you dont want do this for lib.